toolset/benchmark/fortune_html_parser.py

```diff
@@ -74,20 +74,30 @@ def handle_charref(self, name):
     # Not sure why some are escaping '/'
     if val == "47" or val == "047" or val == "x2f":
       self.body.append("/")
+    # "&#40;" is a valid escaping of "(", but
+    # it is not required, so we need to normalize for out
+    # final parse and equality check.
+    if val == "40" or val == "040" or val == "x28":
```

Comment:
Hmm. Very strange. I will do some investigation as to why this might be happening. But otherwise, your pull request looks good!